packages/form-js-viewer/test/spec/render/components/form-fields/Number.spec.js

```diff
@@ -1,5 +1,6 @@
 import {
```

Comment:
Let us add some more tests for the `describe('a11y')` test suite here to ensure we are still green with a decimal number.